### ISSUE TITLE
Re-creating perturbed node detection for finance data

### ISSUE THREAD

**Issue by dplarson** [davidhallac/TVGL#2]
I'm attempting to re-create the result in Figure 5 of the paper (labeled "Perturbed Node Detection for Finance Data"), which shows a spike in the Temporal Deviation (TD) of the finance network on January 27th, 2010 (when Apple announced the iPad). Based on the comments in the code, it appears that the file ``PaperCode/financeInference.py`` should re-create the result. But while the code generates a figure with a Temporal Deviation subplot (y-axis label:``||\Theta_{i} - \Theta_{i-1}||_F``), there doesn't not appear to be a clear spike as in Figure 5:
![tvgl_finance_perturbed_node](https://user-images.githubusercontent.com/1169109/30447845-0664abf2-9942-11e7-8f4d-8d7f16552ef4.png)

Any ideas about this issue? Maybe I've misunderstood what Figure 5 was showing (e.g. it shows the TD ratio described in Section 6.1, instead of the TD).

Thanks!

**Comment by davidhallac** [comment]
Thanks for pointing that out! I just re-ran it, and the error is due to the fact that we had increased the convergence tolerance in the code (but used the stricter tolerance in the paper).

I just pushed the updated file (the only difference is changing the convergence tolerances in lines 41 and 42), and that should generate the same Temporal Deviation plots as in Figure 5 in our paper.

 Let us know if you have any other questions. Thanks again for letting us know!
